### PR TITLE
Sash/Add Deriv.com to group history

### PR DESCRIPTION
**Pull request by sash-binary**

### CODE REVIEW

File: src/templates/static/about/group_history.jsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ const GroupHistory = () => (
             <div className='timeline'>
                 <UL items={[
                     { className: 'year center-text',                                                           text: '2019' },
+                    { className: 'event featured',          header: it.L('Deriv.com launched'),             text: it.L('[_2] was launched as the next-generation online trading platform. [_1] and Deriv collectively handle over 200 million transactions, with over USD 1 billion in turnover a year.', it.website_name, '<a target=\'_blank\' href=\'https://deriv.com\' rel=\'noopener noreferrer\'>Deriv.com</a>') },
```
Comment (negarn) [inline src/templates/static/about/group_history.jsx]: @sash-binary can you fix the indentation of header and text so they match the other lines please?

Comment (sash-binary) [reply src/templates/static/about/group_history.jsx]: updated

File: src/templates/static/about/group_history.jsx [hunk 2]
```diff
@@ -39,6 +39,7 @@ const GroupHistory = () => (
             <div className='timeline'>
                 <UL items={[
                     { className: 'year center-text',                                                           text: '2019' },
+                    { className: 'event featured',       header: it.L('Deriv.com launched'),                   text: it.L('[_2] was launched as the next-generation online trading platform. [_1] and Deriv collectively handle over 200 million transactions, with over USD 1 billion in turnover a year.', it.website_name, '<a target=\'_blank\' href=\'https://deriv.com\' rel=\'noopener noreferrer\'>Deriv.com</a>') },
```
Comment (negarn) [inline src/templates/static/about/group_history.jsx]: IMO it's better to keep the placeholders ordered as 1 and then 2
```suggestion
                    { className: 'event featured',       header: it.L('Deriv.com launched'),                   text: it.L('[_1] was launched as the next-generation online trading platform. [_2] and Deriv collectively handle over 200 million transactions, with over USD 1 billion in turnover a year.', '<a target=\'_blank\' href=\'https://deriv.com\' rel=\'noopener noreferrer\'>Deriv.com</a>', it.website_name) },
```

Comment (sash-binary) [reply src/templates/static/about/group_history.jsx]: updated

**Comment by sash-binary** [comment]
> @sash-binary how about handling the eu client third-party external link?

It turns out Deriv is 3rd party for EU clients and popup must stay

**Pull request merged**